jbmc/src/java_bytecode/java_class_loader.h

```diff
@@ -111,10 +119,21 @@ class java_class_loadert:public messaget
   /// information.
   std::string java_cp_include_files;
 
-  /// List of filesystem paths to .jar files that will be used, in the given
-  /// order, to find and load a class, provided its name (see \ref
-  /// get_parse_tree).
-  std::list<std::string> jar_files;
+  /// An entry in the classpath
+  struct classpath_entryt
+  {
+    using kindt = enum { JAR, DIRECTORY };
```

Comment:
`enum class` ?

Reply:
An enum class provides no obvious benefit here -- the namespace isn't crowded.